CoreFoundation/PlugIn.subproj/CFBundle_Executable.c

```diff
@@ -15,40 +15,37 @@
 #include <dlfcn.h>
 #endif
 
-#if !DEPLOYMENT_RUNTIME_OBJC && !DEPLOYMENT_TARGET_WINDOWS && !DEPLOYMENT_TARGET_ANDROID
+#if FHS_BUNDLES
```

Comment:
While it looks cleaner, the use of explicit platforms was intentional — we need to see what code is called on what paths, and obscuring those platforms makes us no favor. It's long but useful.

(In addition, as a note for the future since we should remove this: we _do not_ introduce macros without prefixes. If we wanted to keep this, it should've been `_CF_…` since it's internal.)

Reply:
I have changed it but added a comment (`/* FHS_BUNDLES */` or `/* FREESTANDING_BUNDLES */`) in each appearance so all cases are easily searchable. They were already out of sync (some of the `FHS_BUNDLES` cases had not the `DEPLOYMENT_TARGET_ANDROID` and some had them).